megamek/src/megamek/server/GameManager.java

```diff
@@ -1090,7 +1090,8 @@ private void resetEntityPhase(GamePhase phase) {
             // reset spotlights
             // If deployment phase, set Searchlight state based on startSearchLightsOn;
             if (phase.isDeployment()) {
-                boolean startSLOn = PreferenceManager.getClientPreferences().getStartSearchlightsOn();
+                boolean startSLOn = PreferenceManager.getClientPreferences().getStartSearchlightsOn()
+                        && game.getPlanetaryConditions().isIlluminationEffective();;
```

Comment:
Think we only need one semi colon at the end of a line. :)

Reply:
Sometimes Idea gets Ideas of its own... -_-;

Fixed.